plugins/notifiers/slack/client.go

```diff
@@ -43,7 +47,12 @@ type Config struct {
 }
 
 func New(config *Config) *notifier {
-	return &notifier{config.AccessToken, map[string]string{}, config.Messages}
+	return &notifier{
```

Comment:
shouldn't we initialized the default template by reading the json here?

Reply:
Done